roles/pulp/tasks/install.yml

```diff
@@ -49,20 +55,30 @@
         system: true
       when: developer_user is not defined
 
+    - name: Add user {{ pulp_user }} to 'pulp' group
```

Comment:
Haven't looked at the details, but would this still happen if `developer_user` is set? Should it add the developer user to the pulp group?

Reply:
We usually set 'developer_user' to the same thing as 'pulp_user'. And that value is usually 'vagrant'.

Reply:
'Usually' sounds like it'd be safer to also do it for `developer_user`. Ansible should be idempotent so it shouldn't hurt when they do match.

Reply:
Just pushed again with this change.